experimentA/idr0045-experimentA-bulkmap-config.yml

```diff
@@ -47,3 +41,8 @@ columns:
         - name: Characteristics [Organism]
           clientname: Organism
           include: yes
+        - name: Term Source 1 Accession
```

Comment:
Looking at other configuration files, what happens if you duplicate the `name` here i.e. `name: Characteristics [Organism]`?

Reply:
Then it would be next to the Organism, but it'll take the value for the `Characteristics [Organism]` field, ie. creates the link`https://www.ncbi.nlm.nih.gov/taxonomy/Mus Musculus` .

Reply:
I see. Then I would default to not exposing the accession number as we have done for previous studies for now.